### PR TITLE
docs(android): correctly document uncaughtException payload

### PR DESCRIPTION
**Pull request by garymathews**
- Fix parity issues between iOS and Android with `uncaughtException` payload

##### TEST CASE
```JS
Ti.App.addEventListener('uncaughtException', e => {
    console.log(JSON.stringify(e, null, 4));
});
throw Error('Exception!');
```
- `column`, `stack` and `nativeStack` should exist on both iOS and Android

[JIRA Ticket](https://jira.appcelerator.org/browse/TIMOB-27632)

### CODE REVIEW

File: apidoc/Titanium/App/App.yml [hunk 1]
```diff
@@ -303,7 +306,35 @@ events:
...
         type: Number
+        deprecated: 
+          since: "9.0.0"
         platforms: [android]
+      - name: jsStack
+        summary: The javascript stack trace of the exception.
+        type: String
+        since: "8.0.0"
+        deprecated: 
+          since: "9.0.0"
```
Comment (sgtcoolguy) [inline apidoc/Titanium/App/App.yml]: add `notes: Use stack property for cross-platform parity`

File: apidoc/Titanium/App/App.yml [hunk 2]
```diff
@@ -303,7 +306,35 @@ events:
       - name: lineOffset
         summary: The offset on the line where the error occurred.
         type: Number
+        deprecated: 
+          since: "9.0.0"
```
Comment (sgtcoolguy) [inline apidoc/Titanium/App/App.yml]: add `notes: Use column property for cross-platform parity`

File: apidoc/Titanium/App/App.yml [hunk 3]
```diff
@@ -303,7 +306,35 @@ events:
...
+        since: "8.0.0"
+        deprecated: 
+          since: "9.0.0"
+        platforms: [android]
+      - name: javaStack
+        summary: The java stack trace of the exception.
+        type: String
+        since: "8.0.0"
+        deprecated: 
+          since: "9.0.0"
```
Comment (sgtcoolguy) [inline apidoc/Titanium/App/App.yml]: add `notes: Use nativeStack property for cross-platform parity`

File: apidoc/Titanium/App/App.yml [hunk 4]
```diff
@@ -303,7 +306,35 @@ events:
...
+        type: String
+        since: "9.0.0"
+      - name: nativeStack
+        summary: The native platform stack trace of the exception.
+        type: String
+        since: "9.0.0"
+      - name: column
+        summary: The column offset on the line where the error occured.
+        type: Number
+        since: "9.0.0"
```
Comment (sgtcoolguy) [inline apidoc/Titanium/App/App.yml]: I assume technically it was present earlier for iOS?

**Review by sgtcoolguy — CHANGES_REQUESTED** [review]
Should include pointers to the "new" properties in the docs.

**Review by sgtcoolguy — APPROVED** [review]

**Comment by lokeshchdhry** [comment]
@garymathews , I don't see `nativeStack` for IOS. Ran it on IOS sim.

**Comment by sgtcoolguy** [comment]
To echo @lokeshchdhry iOS is showing `stackTrace` in this manual test case where you'd expect `nativeStack`.

**Comment by sgtcoolguy** [comment]
I just pushed a fix for the issue @lokeshchdhry and I noted. The uncaughtException now reports the native stack trace as `nativeStack`, not `stackTrace` for the provided test case above. (it also properly provides the `column` and `line` values now too). Ready to re-test for FR @lokeshchdhry

**Comment by lokeshchdhry** [comment]
FR Passed.

`column`, `stack` and `nativeStack` exists on both iOS and Android.

Studio Ver:       6.0.0.201911251516
SDK Ver:          9.0.0 local build
OS Ver:           10.14.5
Xcode Ver:        Xcode 11.3
Appc NPM:         5.0.0-2
Appc CLI:         7.1.2
Daemon Ver:       1.1.3
Ti CLI Ver:       5.2.2
Alloy Ver:        1.14.4
Node Ver:         12.13.1
NPM Ver:          6.12.1
Java Ver:         11.0.1
Android Devices:  ⇨ google Pixel (Android 10)
IOS Simulator:    ⇨ iPhone 11 Pro Max iOS 13.3

**Pull request merged**